### PR TITLE
Add some bit manipulation

### PR DESCRIPTION
**Pull request by danghai**
PR for adding some bit manipulation. 
Relate to #241

### CODE REVIEW

File: bit/has_alternative_bit.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Output: False because the binary representation of 7 is: 111.
+
+Input: 11
+Output: False because the binary representation of 11 is: 1011.
+
+Input: 10
+Output: True because The binary representation of 10 is: 1010.
+"""
+
+def has_alternative_bit(n):
```
Comment (goswami-rahul) [inline bit/has_alternative_bit.py]: This can also be done using bit mask in O(1). Should I commit that algo in this same PR?

**Comment by goswami-rahul** [comment]
@danghai check out another method I added.

**Comment by danghai** [comment]
@goswami-rahul Oh it is better my solution with O(1)

File: bit/has_alternative_bit.py (hunk 1 above)
Comment (danghai) [reply bit/has_alternative_bit.py]: I merge your commit in my RP. Thanks

**Comment by danghai** [comment]
It is good to merge now.

**Pull request merged**